.github/workflows/test.yml

```diff
@@ -27,21 +33,39 @@ jobs:
           node-version: "${{ matrix.node-version }}"
           cache: npm
       - run: npm clean-install
+      - run: npm audit signatures
       - run: npm test
-  test:
+
+  # verify against the node version defined for development in the .nvmrc
+  test_dev:
     runs-on: ubuntu-latest
-    needs: test_matrix
+    timeout-minutes: 5
+
     steps:
-      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
-      - uses: actions/setup-node@5e21ff4d9bc1a8cf6de233a3057d20ec6b3fb69d # v3
+      - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4
+      - run: git config --global user.name github-actions
+      - run: git config --global user.email github-actions@github.com
+      - name: Use Node.js from .nvmrc
+        uses: actions/setup-node@5e21ff4d9bc1a8cf6de233a3057d20ec6b3fb69d # v3
         with:
-          node-version: "lts/*"
+          node-version-file: .nvmrc
```

Comment:
i try to make the nvmrc the source of truth for as much as possible. even more important for apps where supporting a matrix is less of an issue. 

big side benefit of having a job in the workflow that references that file is that renovate updates this file, so it can fail before merge if anything about the project doesnt support that version